contracts/governance/Staking/Staking.sol

```diff
@@ -323,6 +366,24 @@ contract Staking is IStaking, WeightedStaking, ApprovalReceiver {
 		emit TokensWithdrawn(msg.sender, receiver, amount);
 	}
 
+	// @dev withdraws tokens for lock date 2 weeks later than given lock date
+	function _withdrawNext(
+		uint96 amount,
+		uint256 until,
+		address receiver,
+		bool isGovernance
+	) internal {
+		if (msg.sender.isContract()) {
+			uint256 nextLock = until.add(TWO_WEEKS);
+			if (isGovernance || block.timestamp >= nextLock) {
+				uint96 stake = _getPriorUserStakeByDate(msg.sender, nextLock, block.number - 1);
+				if (stake > 1) {
```

Comment:
same here, `_getPriorUserStakeByDate`  returns 0 correctly, so the if-clause should be `if (stake > 0) {`

Reply:
You're right, it doesn't make sense here,
will be checkde in _withdraw anyway.
Changed to 0.